openhtf/core/test_state.py

```diff
@@ -405,7 +408,9 @@ def _is_aborted(self):
     return False
 
   def _outcome_is_failure_exception(self, outcome):
-    for failure_exception in self.failure_exceptions:
+    if not self.test_options.failure_exceptions:
+      return False
```

Comment:
Seems like this isn't needed since `failure_exceptions` is given type `list` in `TestOptions`

Reply:
https://github.com/google/openhtf/pull/774